supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobCachingActorHelper.scala

```diff
@@ -37,20 +31,24 @@ trait JesJobCachingActorHelper extends StandardCachingActorHelper {
   lazy val jesStdoutFile: Path = jesCallPaths.stdout
   lazy val jesStderrFile: Path = jesCallPaths.stderr
   lazy val jesLogFilename: String = jesCallPaths.jesLogFilename
-  lazy val defaultMonitoringOutputPath: Path = callRootPath.resolve(JesMonitoringLogFile)
+
+  lazy val jesMonitoringParamName: String = JesJobPaths.JesMonitoringKey
+  lazy val localMonitoringLogPath: Path = DefaultPathBuilder.get(jesCallPaths.jesMonitoringLogFilename)
+  lazy val localMonitoringScriptPath: Path =  DefaultPathBuilder.get(jesCallPaths.jesMonitoringScriptFilename)
```

Comment:
(continued) and this?

I don't know what the usage differences are, but one is building an absolute path, and the other is building a relative path.

The fact that `localMonitoringScriptPath` is only a relative path and not an absolute path is is causing the test `JesAsyncBackendJobExecutionActor should create a JesFileInput for the monitoring script, when specified` to fail in the `cromwell.backend.impl.jes.JesAsyncBackendJobExecutionActorSpec`.